pkg/api/index.go

```diff
@@ -50,16 +50,22 @@ func (hs *HTTPServer) setIndexViewData(c *contextmodel.ReqContext) (*dtos.IndexV
 	// Locale is used for some number and date/time formatting, whereas language is used just for
 	// translating words in the interface
 	acceptLangHeader := c.Req.Header.Get("Accept-Language")
-	locale := "en-US"
+	locale := ""   // frontend will set the default locale
 	language := "" // frontend will set the default language
 
 	if prefs.JSONData.Language != "" {
 		language = prefs.JSONData.Language
 	}
 
-	if len(acceptLangHeader) > 0 {
-		parts := strings.Split(acceptLangHeader, ",")
-		locale = parts[0]
+	if prefs.JSONData.Locale != "" {
+		locale = prefs.JSONData.Locale
+	} else {
+		if len(acceptLangHeader) > 0 {
+			parts := strings.Split(acceptLangHeader, ",")
+			locale = parts[0]
+		} else {
+			locale = "en-US"
```

Comment:
Since this is the default fallback, maybe you can initiate the variable to this instead?